cmd/config/get/get.go

```diff
@@ -37,12 +38,18 @@ func NewConfigGetCommand() *cobra.Command {
 	return &cobra.Command{
 		Use:   "get [flags] VARIABLE",
 		Short: "Prints the value of a config variable",
-		Long:  "Prints the value of a config variable. See 'rosa config --help' for supported config variables.",
-		Args:  cobra.ExactArgs(1),
-		Run:   run,
+		Long: fmt.Sprintf("Prints the value of a config variable. Supported variables are: %s.",
+			strings.Join(GetAllConfigProperties(), ", ")),
```

Comment:
@tirthct The general pattern seems to be to print supported values on a new line e.g `rosa help config`

Reply:
Changed to print in new line